### ISSUE TITLE
Simultaneous crash for several people using Map Together

### ISSUE THREAD

**Issue by XertroV** [openplanet-nl/openplanet-issues#459]
![image](https://github.com/openplanet-nl/issues/assets/1046448/aceb5ff1-c516-4da1-b185-a1a777410251)

Ville got this crash at the same time as some other people in the room. Is this an assertion that the number if instances of some class is under 1 million? (I suspect I have some issue with the code / memory leak with cursor updates)

**Comment by codecat** [comment]
Yes, there is not a lot I can to fix this, nor can I see why this happened. Would need a minimal reproduce case to see if this is an Openplanet bug or just you having too many references to something in scripts (somehow).
